### ISSUE TITLE
Support case sensitive files

### ISSUE THREAD

**Issue by enoyhs** [srikanth-lingala/zip4j#395]
I have run into a problem where using zip4j to process APK or JAR files which contain case sensitive files, for example, minified APKs contain a lot of resources that differ only by case (`res/aA.xml`, `res/aa.xml`).

As far as I understand zip files should support this or at least there is no explicit requirement for files to be case insensitive. Zip files could actually contain duplicate files, but I don't want/need that.

There is of course a potential issue of trying to extract these zips on case insensitive system (e.g. Windows, OSX), so there might be a need for some warnings or something, but that might be out of scope for this.

Attached is a simple zip containing two files and java code that I would execute to create such a file.
```java
try (ZipFile zipFile = new ZipFile("test.zip")) {
    ZipParameters zp = new ZipParameters();
    zp.setFileNameInZip("case.txt");
    zipFile.addFile("test.txt", zp);
    zp.setFileNameInZip("CASE.txt");
    zipFile.addFile("test.txt", zp);
}
```

[test.zip](https://github.com/srikanth-lingala/zip4j/files/7845977/test.zip)

**Comment by srikanth-lingala** [comment]
This is a good point, but I think it is better to put this as a flag in `ZipParameters`. Something like `ZipParameters.caseSensistiveFileNames` which is false by default, and can be switched on explicitly if needed. Reason behind this thought is that this change might make zip4j backward incompatible. There might be systems integrated with zip4j which rely on the fact that file names are currently insensitive in zip4j and will be replaced if a file is being added which already exists in the zip file.

**Comment by srikanth-lingala** [comment]
I merged [another similar PR](https://github.com/srikanth-lingala/zip4j/pull/396) which does case sensitive file header names comparison by default. After some further considerations, I think it is better to leave use case sensitive comparison by default.

I will include this in the next release. From then on, file header name comparison will be case sensitive.

**Comment by srikanth-lingala** [comment]
Fixed in v2.11.1 released today
